Source/UserManagement/Concepts/Role.cs

```diff
@@ -11,6 +11,14 @@ public enum Role
 
     public static class RoleExtensions
     {
+        public static bool RequiresAgeAndSex(this Role r) =>
```

Comment:
This is purely a style thing, but I prefer to make the enum a class when it starts to have extra info or behaviour.  In the interests of full disclosure, I should note that I'm not a huge fan of enums in general. :-)